ansible/roles/operator-pipeline/templates/openshift/tasks/digest-pinning.yml

```diff
@@ -73,6 +73,9 @@ spec:
           exit 0
         fi
 
+        # configure git to enable operations on unowned repository
+        git config --global --add safe.directory "/workspace/source"
```

Comment:
How about `$(workspaces.source.path)` to not hard code the path on multiple places?

Reply:
Thanks, shame on me - will push update.